common/views/components/WorkCredit/WorkCredit.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import { Fragment } from 'react';
 import type { Work } from '../../../model/work';
-import licenses from '../../../data/licenses';
+import { licenseMap } from '@weco/common/utils/get-license-info';
```

Comment:
I would try keep data and functionality separate, as we will hopefully come from somewhere else at some stage.

How does having `utils/licenses`, `model/licenses`, `data/licenses` feel?

Just a thought.

Reply:
I don't have enough context to understand this comment, but I'm going to merge this so that I can unblock the pipeline stuff if that's ok